### PR TITLE
Update INSTALL_COQIDE.md

### PR DESCRIPTION
**Pull request by benediktahrens**
Point to emacs configuration file template

relates to #1537

### CODE REVIEW

**Comment by DanGrayson** [comment]
Maybe it would be better to say something like "where `<flags>` are the options passed to Coq as per the [Emacs configuration file](./UniMath/.dir-locals.el), which is made from the file `UniMath/.dir-locals.el.in` as a result of running `make`."

That's because `.dir-locals.el` is the emacs configuration file.

**Comment by peterlefanulumsdaine** [comment]
This PR looks good to me, esp with Dan’s suggested tweak

File: INSTALL_COQIDE.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ order to run the program `coqide`, you may use the following command.
 ```bash
 $ sub/coq/bin/coqide <flags>
 ```
-where `<flags>` are the options passed to Coq as per the [Emacs configuration file](./UniMath/.dir-locals.el).
+where `<flags>` are the options passed to Coq as per the [Emacs configuration file](./UniMath/.dir-locals.el.in).
```
Comment (benediktahrens) [inline INSTALL_COQIDE.md:21]:
```suggestion
where `<flags>` are the options passed to Coq as per the [Emacs configuration file](./UniMath/.dir-locals.el), which is made from the file [`UniMath/.dir-locals.el.in`](./UniMath/.dir-locals.el) as a result of running `make`..
```

**Comment by Cypher1** [comment]
Thank you, it should make and I'll get the file needed

**Pull request merged**